### PR TITLE
Add Fahren Lernen

### PR DESCRIPTION
**Pull request by sn00py1310**

### CODE REVIEW

File: img/f/fahren-lernen.de.svg [hunk 1]
```diff
@@ -0,0 +1,8 @@
...
+<svg version="1.0" xmlns="http://www.w3.org/2000/svg" width="180.000000pt" height="180.000000pt" viewBox="0 0 180.000000 180.000000" preserveAspectRatio="xMidYMid meet">
+
+<g transform="translate(0.000000,180.000000) scale(0.100000,-0.100000)" stroke="none" fill="#183e7a">
+<path d="M0 900 l0 -900 900 0 900 0 0 900 0 900 -900 0 -900 0 0 -900z m470 510 l0 -150 -75 0 -75 0 0 150 0 150 75 0 75 0 0 -150z m237 109 l22 -8 -23 -73 c-22 -71 -23 -73 -53 -71 -17 2 -42 7 -56 12 l-24 10 15 76 15 76 41 -7 c23 -3 51 -10 63 -15z m233 -99 c0 -14 -73 -141 -78 -137 -4 1 -25 11 -47 21 -22 10 -41 19 -43 21 -3 3 43 129 53 145 6 9 115 -39 115 -50z m168 -103 c12 -11 22 -24 22 -28 0 -9 -91 -119 -99 -119 -3 1 -23 16 -45 34 l-38 33 43 66 c44 65 44 66 69 50 14 -9 35 -25 48 -36z m167 -159 c177 -197 285 -479 285 -745 l0 -104 -77 3 -78 3 -6 105 c-14 216 -69 381 -188 559 -7 12 -19 6 -61 -29 l-51 -43 -49 54 c-27 30 -49 56 -49 59 1 7 212 189 221 190 3 0 27 -24 53 -52z m-355 -250 c0 -7 -83 -120 -185 -251 -134 -172 -185 -245 -185 -264 0 -42 -37 -106 -75 -130 -98 -64 -235 10 -235 126 0 80 62 149 137 154 36 2 68 24 279 189 132 103 245 187 252 187 6 1 12 -5 12 -11z"/>
+</g>
+</svg>
```
Comment (phallobst) [inline img/f/fahren-lernen.de.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" width="240" height="240" viewBox="0 0 180 180"><path d="M0 90v90h180V0H0v90zm47-51v15H32V24h15v15zm23.7-10.9 2.2.8-2.3 7.3c-2.2 7.1-2.3 7.3-5.3 7.1-1.7-.2-4.2-.7-5.6-1.2l-2.4-1 1.5-7.6 1.5-7.6 4.1.7c2.3.3 5.1 1 6.3 1.5zM94 38c0 1.4-7.3 14.1-7.8 13.7-.4-.1-2.5-1.1-4.7-2.1s-4.1-1.9-4.3-2.1c-.3-.3 4.3-12.9 5.3-14.5.6-.9 11.5 3.9 11.5 5zm16.8 10.3c1.2 1.1 2.2 2.4 2.2 2.8 0 .9-9.1 11.9-9.9 11.9-.3-.1-2.3-1.6-4.5-3.4l-3.8-3.3 4.3-6.6c4.4-6.5 4.4-6.6 6.9-5 1.4.9 3.5 2.5 4.8 3.6zm16.7 15.9c17.7 19.7 28.5 47.9 28.5 74.5v10.4l-7.7-.3-7.8-.3-.6-10.5c-1.4-21.6-6.9-38.1-18.8-55.9-.7-1.2-1.9-.6-6.1 2.9l-5.1 4.3-4.9-5.4c-2.7-3-4.9-5.6-4.9-5.9.1-.7 21.2-18.9 22.1-19 .3 0 2.7 2.4 5.3 5.2zM92 89.2c0 .7-8.3 12-18.5 25.1C60.1 131.5 55 138.8 55 140.7c0 4.2-3.7 10.6-7.5 13-9.8 6.4-23.5-1-23.5-12.6 0-8 6.2-14.9 13.7-15.4 3.6-.2 6.8-2.4 27.9-18.9 13.2-10.3 24.5-18.7 25.2-18.7.6-.1 1.2.5 1.2 1.1z" fill="#183e7a"/></svg>
```

**Review by phallobst — CHANGES_REQUESTED** [review]

**Review by sn00py1310 — COMMENTED** [review]
Changes made to SVG file

**Review by phallobst — APPROVED** [review]

**Comment by Carlgo11** [comment]
So what is Fahren Lernen?
A drivers school? Transports agency?

**Comment by phallobst** [comment]
You can practice for the written part of the drivers license test on that page.

**Review by Carlgo11 — APPROVED** [review]

**Pull request merged**